mappings/net/minecraft/server/dedicated/ServerMBean.mapping

```diff
@@ -0,0 +1,45 @@
+CLASS net/minecraft/class_5169 net/minecraft/server/dedicated/ServerMBean
+	COMMENT A dynamic management object for a Minecraft Server.
+	COMMENT
+	COMMENT <p>It exposes the average tick time and the historical tick times of the
+	COMMENT Minecraft Server.</p>
+	COMMENT
+	COMMENT @see javax.management.DynamicMBean
+	FIELD field_23970 LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD field_23971 server Lnet/minecraft/server/MinecraftServer;
+	FIELD field_23972 beanInfo Ljavax/management/MBeanInfo;
+	FIELD field_23973 entries Ljava/util/Map;
+	METHOD <init> (Lnet/minecraft/server/MinecraftServer;)V
+		ARG 1 server
+	METHOD getAttribute (Ljava/lang/String;)Ljava/lang/Object;
+		ARG 1 attribute
+	METHOD getAttributes ([Ljava/lang/String;)Ljavax/management/AttributeList;
+		ARG 1 attributes
```

Comment:
You named the setter parameters the same as that, but those ones aren't javax.management.Attribute. Maybe `attributeName` would be better?

Reply:
these names are copied from jdk's names.

Reply:
fair